src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -136,68 +129,268 @@
     private static final double LOG_ODDS_THRESHOLD = 3.0;
 
     /** Metrics for evaluating the performance of whole genome sequencing experiments. */
-    public static class WgsMetrics extends MetricBase {
+    public static class WgsMetrics extends MergeableMetricBase {
+
+        /** The intervals over which this metric was computed. */
+        @NoMerging
+        protected IntervalList intervals;
```

Comment:
I"m confused. didn't you say that you do not want to include this here? 

looks like you are merging this manually...so why "NoMerging"?